components/suggestion/search_user_provider.tsx

```diff
@@ -75,20 +81,21 @@ class SearchUserSuggestion extends Suggestion {
 }
 
 export default class SearchUserProvider extends Provider {
-    constructor(userSearchFunc) {
+    autocompleteUsersInTeam: (username: string) => any;
```

Comment:
Due to type mismatching, I am unable to find any solution which can replace 'any' here .